agent/discovery/query_fetcher_v1_ce.go

```diff
@@ -15,7 +15,7 @@ func (f *V1DataFetcher) NormalizeRequest(req *QueryPayload) {
 }
 
 func validateEnterpriseTenancy(req QueryTenancy) error {
-	if req.Namespace != "" || req.Partition != acl.DefaultPartitionName {
+	if !(req.Namespace == "" || req.Namespace == acl.DefaultNamespaceName) || !(req.Partition == acl.DefaultPartitionName || req.Partition == "default") {
```

Comment:
~ It’d be great to use a constant here for the last value if possible, maybe we should introduce or expose one if there’s nothing available?

Reply:
Added, LMK what you think - seems to me like the safer constant to literal is `""`, and we should use a constant at least for non-empty values

Reply:
i already did a different change because I was doing this locally and did not realize you pushed a change..  we don't want to discovery or dns to have a dependency on `api`,so I think keeping things in `acl` is safer because it already imports that.  What do you think of my recent change?

Reply:
That's totally fair - I like the new version, thank you for going the extra mile on this